wiki/Glossary/en.md

```diff
@@ -788,7 +793,7 @@ Star has four meanings:
 
 Star priority has two meanings:
 
-1. Star priority is a method of helping Beatmap Nominators to find beatmaps worthy of bubbling and later ranking.
+1. A method of helping Beatmap Nominators to find beatmaps worthy of bubbling and later ranking.
    - The star priority of a beatmap is determined by how many mod posts that have gained kudosu in the beatmap's thread, the sum of kudosu stars the map set has received.
    - A kudosu star can be given by any user to any map, though shooting a kudosu star comes at the price of one kudosu.
    - The conversion rate was formerly three kudosu for one kudosu star that increased the Star Priority by two.
```

Comment:
```
   - At least 12 star priority is needed for a beatmap to be bubbled or ranked.
```
sidenote: not sure if 12 SP to rank (or bubble) is still a thing and/or if bubbled is still a thing